lib/car.rb

```diff
@@ -0,0 +1,24 @@
+class Car
+  attr_reader :car, :make, :model, :monthly_payment,
+  :loan_length, :color
+  def initialize(make, model, loan_length)
+    @car = car
+    @make = "Ford"
+    @model = "Mustang"
```

Comment:
I would not initialize variables in this way; they should instead be initalized to the parameters in your `#initialize` method. In other words: 
```
def initialize(make, model, loan_length)
     @make = make
     @model = model
     ...
end
```
Otherwise, you are not actually using the values that you pass in when you call `Car.new("Ford Mustang", 1500, 36)`. In addition, the `model` variable actually is equal to `1500` in this sense - so the order in which you pass in data is very important!